synapse/event_auth.py

```diff
@@ -409,7 +409,7 @@ def _can_send_event(event: EventBase, auth_events: StateMap[EventBase]) -> bool:
     if user_level < send_level:
         raise AuthError(
             403,
-            "You don't have permission to post that to the room. "
+            ("You don't have permission to post that %s to the room. " % event.type)
```

Comment:
```suggestion
            ("You don't have permission to post %s to the room. " % (event.type,))
```

prefer to pass an explicit tuple to %.